[2022-08-21 12:35:28,293] {processor.py:163} INFO - Started process (PID=72494) to work on /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2022-08-21 12:35:28,293] {processor.py:620} INFO - Processing file /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py for tasks to queue
[2022-08-21 12:35:28,294] {logging_mixin.py:109} INFO - [2022-08-21 12:35:28,294] {dagbag.py:500} INFO - Filling up the DagBag from /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2022-08-21 12:35:28,314] {processor.py:632} INFO - DAG(s) dict_keys(['tutorial_taskflow_api_etl']) retrieved from /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2022-08-21 12:35:28,319] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 12:35:28,600] {logging_mixin.py:109} INFO - [2022-08-21 12:35:28,600] {dag.py:2389} INFO - Sync 1 DAGs
[2022-08-21 12:35:28,610] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 12:35:28,615] {logging_mixin.py:109} INFO - [2022-08-21 12:35:28,615] {dag.py:2914} INFO - Setting next_dagrun for tutorial_taskflow_api_etl to None
[2022-08-21 12:35:28,632] {processor.py:171} INFO - Processing /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py took 0.341 seconds
[2022-08-21 12:36:03,567] {processor.py:163} INFO - Started process (PID=72642) to work on /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2022-08-21 12:36:03,568] {processor.py:620} INFO - Processing file /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py for tasks to queue
[2022-08-21 12:36:03,568] {logging_mixin.py:109} INFO - [2022-08-21 12:36:03,568] {dagbag.py:500} INFO - Filling up the DagBag from /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2022-08-21 12:36:03,589] {processor.py:632} INFO - DAG(s) dict_keys(['tutorial_taskflow_api_etl']) retrieved from /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2022-08-21 12:36:03,593] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 12:36:03,875] {logging_mixin.py:109} INFO - [2022-08-21 12:36:03,875] {dag.py:2389} INFO - Sync 1 DAGs
[2022-08-21 12:36:03,885] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 12:36:03,891] {logging_mixin.py:109} INFO - [2022-08-21 12:36:03,891] {dag.py:2914} INFO - Setting next_dagrun for tutorial_taskflow_api_etl to None
[2022-08-21 12:36:03,904] {processor.py:171} INFO - Processing /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py took 0.339 seconds
[2022-08-21 12:36:38,820] {processor.py:163} INFO - Started process (PID=72676) to work on /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2022-08-21 12:36:38,821] {processor.py:620} INFO - Processing file /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py for tasks to queue
[2022-08-21 12:36:38,821] {logging_mixin.py:109} INFO - [2022-08-21 12:36:38,821] {dagbag.py:500} INFO - Filling up the DagBag from /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2022-08-21 12:36:38,841] {processor.py:632} INFO - DAG(s) dict_keys(['tutorial_taskflow_api_etl']) retrieved from /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2022-08-21 12:36:38,846] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 12:36:39,112] {logging_mixin.py:109} INFO - [2022-08-21 12:36:39,112] {dag.py:2389} INFO - Sync 1 DAGs
[2022-08-21 12:36:39,121] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 12:36:39,126] {logging_mixin.py:109} INFO - [2022-08-21 12:36:39,126] {dag.py:2914} INFO - Setting next_dagrun for tutorial_taskflow_api_etl to None
[2022-08-21 12:36:39,138] {processor.py:171} INFO - Processing /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py took 0.319 seconds
[2022-08-21 12:37:25,151] {processor.py:163} INFO - Started process (PID=72933) to work on /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2022-08-21 12:37:25,152] {processor.py:620} INFO - Processing file /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py for tasks to queue
[2022-08-21 12:37:25,152] {logging_mixin.py:109} INFO - [2022-08-21 12:37:25,152] {dagbag.py:500} INFO - Filling up the DagBag from /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2022-08-21 12:37:25,173] {processor.py:632} INFO - DAG(s) dict_keys(['tutorial_taskflow_api_etl']) retrieved from /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2022-08-21 12:37:25,177] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 12:37:25,442] {logging_mixin.py:109} INFO - [2022-08-21 12:37:25,441] {dag.py:2389} INFO - Sync 1 DAGs
[2022-08-21 12:37:25,451] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 12:37:25,455] {logging_mixin.py:109} INFO - [2022-08-21 12:37:25,455] {dag.py:2914} INFO - Setting next_dagrun for tutorial_taskflow_api_etl to None
[2022-08-21 12:37:25,467] {processor.py:171} INFO - Processing /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py took 0.318 seconds
[2022-08-21 12:38:00,395] {processor.py:163} INFO - Started process (PID=73094) to work on /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2022-08-21 12:38:00,396] {processor.py:620} INFO - Processing file /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py for tasks to queue
[2022-08-21 12:38:00,396] {logging_mixin.py:109} INFO - [2022-08-21 12:38:00,396] {dagbag.py:500} INFO - Filling up the DagBag from /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2022-08-21 12:38:00,416] {processor.py:632} INFO - DAG(s) dict_keys(['tutorial_taskflow_api_etl']) retrieved from /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2022-08-21 12:38:00,421] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 12:38:00,681] {logging_mixin.py:109} INFO - [2022-08-21 12:38:00,681] {dag.py:2389} INFO - Sync 1 DAGs
[2022-08-21 12:38:00,690] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 12:38:00,695] {logging_mixin.py:109} INFO - [2022-08-21 12:38:00,695] {dag.py:2914} INFO - Setting next_dagrun for tutorial_taskflow_api_etl to None
[2022-08-21 12:38:00,708] {processor.py:171} INFO - Processing /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py took 0.315 seconds
[2022-08-21 12:38:35,648] {processor.py:163} INFO - Started process (PID=73129) to work on /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2022-08-21 12:38:35,648] {processor.py:620} INFO - Processing file /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py for tasks to queue
[2022-08-21 12:38:35,649] {logging_mixin.py:109} INFO - [2022-08-21 12:38:35,649] {dagbag.py:500} INFO - Filling up the DagBag from /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2022-08-21 12:38:35,671] {processor.py:632} INFO - DAG(s) dict_keys(['tutorial_taskflow_api_etl']) retrieved from /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2022-08-21 12:38:35,675] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 12:38:35,948] {logging_mixin.py:109} INFO - [2022-08-21 12:38:35,948] {dag.py:2389} INFO - Sync 1 DAGs
[2022-08-21 12:38:35,957] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 12:38:35,962] {logging_mixin.py:109} INFO - [2022-08-21 12:38:35,962] {dag.py:2914} INFO - Setting next_dagrun for tutorial_taskflow_api_etl to None
[2022-08-21 12:38:35,974] {processor.py:171} INFO - Processing /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py took 0.328 seconds
[2022-08-21 12:39:10,327] {processor.py:163} INFO - Started process (PID=73223) to work on /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2022-08-21 12:39:10,328] {processor.py:620} INFO - Processing file /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py for tasks to queue
[2022-08-21 12:39:10,328] {logging_mixin.py:109} INFO - [2022-08-21 12:39:10,328] {dagbag.py:500} INFO - Filling up the DagBag from /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2022-08-21 12:39:10,352] {processor.py:632} INFO - DAG(s) dict_keys(['tutorial_taskflow_api_etl']) retrieved from /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2022-08-21 12:39:10,356] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 12:39:10,614] {logging_mixin.py:109} INFO - [2022-08-21 12:39:10,614] {dag.py:2389} INFO - Sync 1 DAGs
[2022-08-21 12:39:10,623] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 12:39:10,628] {logging_mixin.py:109} INFO - [2022-08-21 12:39:10,628] {dag.py:2914} INFO - Setting next_dagrun for tutorial_taskflow_api_etl to None
[2022-08-21 12:39:10,642] {processor.py:171} INFO - Processing /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py took 0.317 seconds
[2022-08-21 12:39:46,143] {processor.py:163} INFO - Started process (PID=73256) to work on /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2022-08-21 12:39:46,144] {processor.py:620} INFO - Processing file /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py for tasks to queue
[2022-08-21 12:39:46,144] {logging_mixin.py:109} INFO - [2022-08-21 12:39:46,144] {dagbag.py:500} INFO - Filling up the DagBag from /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2022-08-21 12:39:46,164] {processor.py:632} INFO - DAG(s) dict_keys(['tutorial_taskflow_api_etl']) retrieved from /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2022-08-21 12:39:46,168] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 12:39:46,427] {logging_mixin.py:109} INFO - [2022-08-21 12:39:46,427] {dag.py:2389} INFO - Sync 1 DAGs
[2022-08-21 12:39:46,436] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 12:39:46,441] {logging_mixin.py:109} INFO - [2022-08-21 12:39:46,441] {dag.py:2914} INFO - Setting next_dagrun for tutorial_taskflow_api_etl to None
[2022-08-21 12:39:46,455] {processor.py:171} INFO - Processing /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py took 0.315 seconds
[2022-08-21 12:40:19,289] {processor.py:163} INFO - Started process (PID=73290) to work on /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2022-08-21 12:40:19,290] {processor.py:620} INFO - Processing file /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py for tasks to queue
[2022-08-21 12:40:19,290] {logging_mixin.py:109} INFO - [2022-08-21 12:40:19,290] {dagbag.py:500} INFO - Filling up the DagBag from /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2022-08-21 12:40:19,312] {processor.py:632} INFO - DAG(s) dict_keys(['tutorial_taskflow_api_etl']) retrieved from /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2022-08-21 12:40:19,317] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 12:40:19,594] {logging_mixin.py:109} INFO - [2022-08-21 12:40:19,594] {dag.py:2389} INFO - Sync 1 DAGs
[2022-08-21 12:40:19,605] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 12:40:19,611] {logging_mixin.py:109} INFO - [2022-08-21 12:40:19,611] {dag.py:2914} INFO - Setting next_dagrun for tutorial_taskflow_api_etl to None
[2022-08-21 12:40:19,625] {processor.py:171} INFO - Processing /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py took 0.338 seconds
[2022-08-21 12:40:55,112] {processor.py:163} INFO - Started process (PID=73371) to work on /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2022-08-21 12:40:55,112] {processor.py:620} INFO - Processing file /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py for tasks to queue
[2022-08-21 12:40:55,113] {logging_mixin.py:109} INFO - [2022-08-21 12:40:55,113] {dagbag.py:500} INFO - Filling up the DagBag from /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2022-08-21 12:40:55,133] {processor.py:632} INFO - DAG(s) dict_keys(['tutorial_taskflow_api_etl']) retrieved from /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2022-08-21 12:40:55,137] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 12:40:55,401] {logging_mixin.py:109} INFO - [2022-08-21 12:40:55,401] {dag.py:2389} INFO - Sync 1 DAGs
[2022-08-21 12:40:55,411] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 12:40:55,416] {logging_mixin.py:109} INFO - [2022-08-21 12:40:55,416] {dag.py:2914} INFO - Setting next_dagrun for tutorial_taskflow_api_etl to None
[2022-08-21 12:40:55,429] {processor.py:171} INFO - Processing /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py took 0.319 seconds
[2022-08-21 12:41:25,975] {processor.py:163} INFO - Started process (PID=73462) to work on /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2022-08-21 12:41:25,975] {processor.py:620} INFO - Processing file /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py for tasks to queue
[2022-08-21 12:41:25,976] {logging_mixin.py:109} INFO - [2022-08-21 12:41:25,976] {dagbag.py:500} INFO - Filling up the DagBag from /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2022-08-21 12:41:25,996] {processor.py:632} INFO - DAG(s) dict_keys(['tutorial_taskflow_api_etl']) retrieved from /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2022-08-21 12:41:26,001] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 12:41:26,261] {logging_mixin.py:109} INFO - [2022-08-21 12:41:26,261] {dag.py:2389} INFO - Sync 1 DAGs
[2022-08-21 12:41:26,272] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 12:41:26,277] {logging_mixin.py:109} INFO - [2022-08-21 12:41:26,277] {dag.py:2914} INFO - Setting next_dagrun for tutorial_taskflow_api_etl to None
[2022-08-21 12:41:26,290] {processor.py:171} INFO - Processing /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py took 0.317 seconds
[2022-08-21 12:41:58,652] {processor.py:163} INFO - Started process (PID=73506) to work on /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2022-08-21 12:41:58,652] {processor.py:620} INFO - Processing file /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py for tasks to queue
[2022-08-21 12:41:58,653] {logging_mixin.py:109} INFO - [2022-08-21 12:41:58,653] {dagbag.py:500} INFO - Filling up the DagBag from /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2022-08-21 12:41:58,673] {processor.py:632} INFO - DAG(s) dict_keys(['tutorial_taskflow_api_etl']) retrieved from /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2022-08-21 12:41:58,677] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 12:41:58,941] {logging_mixin.py:109} INFO - [2022-08-21 12:41:58,941] {dag.py:2389} INFO - Sync 1 DAGs
[2022-08-21 12:41:58,951] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 12:41:58,956] {logging_mixin.py:109} INFO - [2022-08-21 12:41:58,956] {dag.py:2914} INFO - Setting next_dagrun for tutorial_taskflow_api_etl to None
[2022-08-21 12:41:58,969] {processor.py:171} INFO - Processing /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py took 0.319 seconds
[2022-08-21 12:42:34,495] {processor.py:163} INFO - Started process (PID=73676) to work on /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2022-08-21 12:42:34,495] {processor.py:620} INFO - Processing file /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py for tasks to queue
[2022-08-21 12:42:34,496] {logging_mixin.py:109} INFO - [2022-08-21 12:42:34,496] {dagbag.py:500} INFO - Filling up the DagBag from /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2022-08-21 12:42:34,518] {processor.py:632} INFO - DAG(s) dict_keys(['tutorial_taskflow_api_etl']) retrieved from /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2022-08-21 12:42:34,522] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 12:42:34,799] {logging_mixin.py:109} INFO - [2022-08-21 12:42:34,799] {dag.py:2389} INFO - Sync 1 DAGs
[2022-08-21 12:42:34,811] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 12:42:34,816] {logging_mixin.py:109} INFO - [2022-08-21 12:42:34,816] {dag.py:2914} INFO - Setting next_dagrun for tutorial_taskflow_api_etl to None
[2022-08-21 12:42:34,830] {processor.py:171} INFO - Processing /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py took 0.338 seconds
[2022-08-21 12:43:08,046] {processor.py:163} INFO - Started process (PID=73769) to work on /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2022-08-21 12:43:08,046] {processor.py:620} INFO - Processing file /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py for tasks to queue
[2022-08-21 12:43:08,047] {logging_mixin.py:109} INFO - [2022-08-21 12:43:08,047] {dagbag.py:500} INFO - Filling up the DagBag from /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2022-08-21 12:43:08,068] {processor.py:632} INFO - DAG(s) dict_keys(['tutorial_taskflow_api_etl']) retrieved from /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2022-08-21 12:43:08,072] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 12:43:08,332] {logging_mixin.py:109} INFO - [2022-08-21 12:43:08,332] {dag.py:2389} INFO - Sync 1 DAGs
[2022-08-21 12:43:08,341] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 12:43:08,346] {logging_mixin.py:109} INFO - [2022-08-21 12:43:08,346] {dag.py:2914} INFO - Setting next_dagrun for tutorial_taskflow_api_etl to None
[2022-08-21 12:43:08,359] {processor.py:171} INFO - Processing /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py took 0.315 seconds
[2022-08-21 12:43:44,157] {processor.py:163} INFO - Started process (PID=73909) to work on /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2022-08-21 12:43:44,158] {processor.py:620} INFO - Processing file /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py for tasks to queue
[2022-08-21 12:43:44,158] {logging_mixin.py:109} INFO - [2022-08-21 12:43:44,158] {dagbag.py:500} INFO - Filling up the DagBag from /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2022-08-21 12:43:44,178] {processor.py:632} INFO - DAG(s) dict_keys(['tutorial_taskflow_api_etl']) retrieved from /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2022-08-21 12:43:44,182] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 12:43:44,442] {logging_mixin.py:109} INFO - [2022-08-21 12:43:44,442] {dag.py:2389} INFO - Sync 1 DAGs
[2022-08-21 12:43:44,451] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 12:43:44,456] {logging_mixin.py:109} INFO - [2022-08-21 12:43:44,456] {dag.py:2914} INFO - Setting next_dagrun for tutorial_taskflow_api_etl to None
[2022-08-21 12:43:44,468] {processor.py:171} INFO - Processing /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py took 0.313 seconds
[2022-08-21 12:44:20,562] {processor.py:163} INFO - Started process (PID=74156) to work on /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2022-08-21 12:44:20,563] {processor.py:620} INFO - Processing file /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py for tasks to queue
[2022-08-21 12:44:20,563] {logging_mixin.py:109} INFO - [2022-08-21 12:44:20,563] {dagbag.py:500} INFO - Filling up the DagBag from /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2022-08-21 12:44:20,584] {processor.py:632} INFO - DAG(s) dict_keys(['tutorial_taskflow_api_etl']) retrieved from /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2022-08-21 12:44:20,588] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 12:44:20,872] {logging_mixin.py:109} INFO - [2022-08-21 12:44:20,872] {dag.py:2389} INFO - Sync 1 DAGs
[2022-08-21 12:44:20,882] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 12:44:20,887] {logging_mixin.py:109} INFO - [2022-08-21 12:44:20,887] {dag.py:2914} INFO - Setting next_dagrun for tutorial_taskflow_api_etl to None
[2022-08-21 12:44:20,901] {processor.py:171} INFO - Processing /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py took 0.341 seconds
[2022-08-21 12:44:55,537] {processor.py:163} INFO - Started process (PID=74244) to work on /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2022-08-21 12:44:55,537] {processor.py:620} INFO - Processing file /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py for tasks to queue
[2022-08-21 12:44:55,538] {logging_mixin.py:109} INFO - [2022-08-21 12:44:55,538] {dagbag.py:500} INFO - Filling up the DagBag from /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2022-08-21 12:44:55,558] {processor.py:632} INFO - DAG(s) dict_keys(['tutorial_taskflow_api_etl']) retrieved from /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2022-08-21 12:44:55,563] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 12:44:55,825] {logging_mixin.py:109} INFO - [2022-08-21 12:44:55,824] {dag.py:2389} INFO - Sync 1 DAGs
[2022-08-21 12:44:55,834] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 12:44:55,838] {logging_mixin.py:109} INFO - [2022-08-21 12:44:55,838] {dag.py:2914} INFO - Setting next_dagrun for tutorial_taskflow_api_etl to None
[2022-08-21 12:44:55,851] {processor.py:171} INFO - Processing /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py took 0.317 seconds
[2022-08-21 12:45:30,222] {processor.py:163} INFO - Started process (PID=74653) to work on /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2022-08-21 12:45:30,222] {processor.py:620} INFO - Processing file /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py for tasks to queue
[2022-08-21 12:45:30,223] {logging_mixin.py:109} INFO - [2022-08-21 12:45:30,223] {dagbag.py:500} INFO - Filling up the DagBag from /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2022-08-21 12:45:30,244] {processor.py:632} INFO - DAG(s) dict_keys(['tutorial_taskflow_api_etl']) retrieved from /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2022-08-21 12:45:30,248] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 12:45:30,514] {logging_mixin.py:109} INFO - [2022-08-21 12:45:30,514] {dag.py:2389} INFO - Sync 1 DAGs
[2022-08-21 12:45:30,523] {logging_mixin.py:109} WARNING - /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-08-21 12:45:30,528] {logging_mixin.py:109} INFO - [2022-08-21 12:45:30,528] {dag.py:2914} INFO - Setting next_dagrun for tutorial_taskflow_api_etl to None
[2022-08-21 12:45:30,540] {processor.py:171} INFO - Processing /home/ubuntu/anaconda3/lib/python3.9/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py took 0.321 seconds
